modules/oke/cluster.tf

```diff
@@ -12,6 +12,13 @@ resource "oci_containerengine_cluster" "k8s_cluster" {
     subnet_id            = var.oke_cluster.cluster_subnets["cp"]
   }
 
+  image_policy_config {
+    is_policy_enabled = var.oke_cluster.use_signed_images
+    key_details {
+      kms_key_id = var.oke_cluster.use_signed_images == true ? var.oke_cluster.kms_key_id : null
```

Comment:
The key used for signed images can be different from the one used to encrypt etcd, so users must be able to specify each separately. 

Add 1 more variable to allow the user to specify the key for signed images and then pass it along to the oke module and eventually to this line of code.

Reply:
Please add documentation for the new variable in terraform options as well.

Reply:
New variable signedimage_key_id has been added. Doc has been updated as well.

Reply:
1. Please rename to image_signing_keys.
2. Sorry, I just noticed this: You can add multiple keys for image signing. So, this should be a list of key ids.
3. Add a dynamic and iterator so you can iterate over the list of key ids

Reply:
Looks like it does not support list. Inappropriate value for attribute "kms_key_id": string required.

Reply:
Try putting a dynamic at key_details level  and then add an iterator?

Reply:
Changes done please review. I have updated the default node_pool_os_version to 7.9 in tfvars.example as well.